integrations/jenkins/Jenkinsfile

```diff
@@ -143,6 +143,12 @@ pipeline {
             choices: ['ScanCode', 'Askalono', 'BoyterLc', 'Licensee', 'ScanOSS', '<NONE>']
         )
 
+        booleanParam(
+            name: 'USE_CLEARLY_DEFINED_SCAN_STORAGE',
+            description: 'Use the ClearlyDefined service to retrieve existing scan results.',
```

Comment:
Should this mention that enabling this option overrides any storage configuration from `config.yml`?

Reply:
Yes, will do. I first researched whether there's a way in Hoplite to merge lists from different property sources, but [this](https://github.com/sksamuel/hoplite#cascading-config) explicitly says that "List's cannot be merged".